apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/R4PatientResourceProviderE2E.java

```diff
@@ -1,931 +0,0 @@
-   * bene id in either the Beneficiaries and/or BeneficiariesHistory table.
-   */
-  @Test
-  public void searchForExistingPatientByMbiHashWithBeneDupes() {
-    List<Object> loadedRecords =
-        ServerTestUtils.get()
-            .loadData(Arrays.asList(StaticRifResourceGroup.SAMPLE_A.getResources()));
-
-    // load additional Beneficiary and Beneficiary History records for
-    // testing
-    loadedRecords.addAll(
-        ServerTestUtils.get()
-            .loadData(Arrays.asList(StaticRifResourceGroup.SAMPLE_HICN_MULT_BENES.getResources())));
-
-    IGenericClient fhirClient = ServerTestUtils.get().createFhirClientV2();
-
-    Stream<Beneficiary> beneficiariesStream =
-        loadedRecords.stream().filter(r -> r instanceof Beneficiary).map(r -> (Beneficiary) r);
-    List<Beneficiary> beneficiariesList = beneficiariesStream.collect(Collectors.toList());
-
-    Stream<BeneficiaryHistory> beneficiariesHistoryStream =
-        loadedRecords.stream()
-            .filter(r -> r instanceof BeneficiaryHistory)
-            .map(r -> (BeneficiaryHistory) r);
-    List<BeneficiaryHistory> beneficiariesHistoryList =
-        beneficiariesHistoryStream.collect(Collectors.toList());
-
-    boolean useMbiFromBeneficiaryTable;
-    boolean expectsSingleBeneMatch;
-
-    /*
-     * The following scenario tests when the same mbi is in the
-     * Beneficiaries table but points to different bene ids.
-     */
-    useMbiFromBeneficiaryTable = true;
-    expectsSingleBeneMatch = false;
-    assertPatientByHashTypeMatch(
-        fhirClient,
-        beneficiariesList,
-        beneficiariesHistoryList,
-        567834L,
-        "3456789",
-        useMbiFromBeneficiaryTable,
-        expectsSingleBeneMatch,
-        standardExpectedHistoricalMbis);
-
-    /*
-     * The following scenario tests when only one mbi is in the
-     * Beneficiaries table.
-     */
-    useMbiFromBeneficiaryTable = true;
-    expectsSingleBeneMatch = true;
-    assertPatientByHashTypeMatch(
-        fhirClient,
-        beneficiariesList,
-        beneficiariesHistoryList,
-        -123456L,
-        "3456789N",
-        useMbiFromBeneficiaryTable,
-        expectsSingleBeneMatch,
-        new ArrayList<>());
-
-    /*
-     * The following scenario tests when the same mbi is in the
-     * Beneficiaries and also in the BeneficiariesHistory table. The bene id
-     * is different between the tables so the bene record from the
-     * Beneficiaries table should be used.
-     *
-     * bene id=BENE1234 mbi=SAMEMBI rfrnc_yr=2019 should be pulled back.
-     */
-    useMbiFromBeneficiaryTable = true;
-    expectsSingleBeneMatch = false;
-    assertPatientByHashTypeMatch(
-        fhirClient,
-        beneficiariesList,
-        beneficiariesHistoryList,
-        1234L,
-        "SAMEMBI",
-        useMbiFromBeneficiaryTable,
-        expectsSingleBeneMatch,
-        List.of("HISTMBI"));
```

Comment:
This was the test case in question mentioned in the data adjustment. Since "expectsSingleBeneMatch" was set to false, it expected an exception, which was being raised by the data issue, and did not test what the comment above suggested.